app/views/posts/index.html.erb

```diff
@@ -1,2 +1,43 @@
-<h1> Post </h1>
-<p> Lists of all posts </p>
\ No newline at end of file
+<div class='center'>
+    <div class='flex '>
+        <div> <%= image_tag @user.Photo %> </div>
+        <div class='border'>
+            <%= @user.Name %>
+            <div class='bottom_flex'>
+                <%= @user.PostCounter %>
+            </div>
+        </div>
+            </div>
+                    <% @posts.each do |post| %>
+ 
+                       <div class='borBtm'>
+                       <div class='border'>
+                            <div>
+                            <p> post </p>
+                                <%= link_to post.title, user_post_path(@user.id, post.id)%>
+                            </div>
+                            <div>
+                                <%= post.CommentCounter %>
+                            </div>
+                            <div>
+                                <%= post.LikesCounter %>
+                            </div>
+                        </div>
+                        <% if post.five_recent_comments.count.zero? %>
+                        <p class='border'> No comments on this post yet <p>
+                        <% else %>
+                        <div class='border'>
+                            <% post.five_recent_comments.each do |post| %>
+                                <div>
+                                    <%= post.text %>
+                                </div>
+                            <% end %>
+                            <% end %>
+                        </div>
+                        <% %>
+                        </div>
+                    <% end %>
+                </div>
+
+    </div>
+</div>
```

Comment:
- [ ] The post-index page, as per the wireframe, it should show the number of comments and likes. In your application, it does show some numbers, but those numbers should correspond to comments and likes which is missing in your case.
- [ ] It shoud show the username of the user who left the comment. 
- [ ] It should show what number each post is.
Kindly fix these issues.

| Your design  | Wireframe |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/218249659-22c0ab9e-0a96-4202-bdf9-ac5adbf56a7f.png) | ![image](https://user-images.githubusercontent.com/33268927/218249740-0d38f934-6c6a-482a-86a9-2fbe1dce7669.png) |

Reply:
I have fixed every issue stated